repository.json

```diff
@@ -675,6 +675,56 @@
 				}
 			]
 		},
+		{
+			"name": "llm_runner",
+			"description": "Sublime LLM Communication Core",
+			"author": "Yaroslav Ya",
+			"issues": "https://github.com/yaroslavyaroslav/llm_runner/issues",
+			"releases": [
+				{
+					"base": "https://pypi.org/project/rust-helper",
```

Comment:
URL?

Reply:
Oh, my bad, will fix it later today, sorry